### PR TITLE
Ema/fix other regression

### PR DESCRIPTION
**Pull request by GioSensation**
**Asana:** https://app.asana.com/0/0/1201338885189703/f
**Reviewer:** @jonathanKingston 

## Description

Add a few bug fixes and improvements to field detection and classification. Remember that you can skip the initial file `dist/autofill.js`. That's just the compiled code.

### CODE REVIEW

File: src/Form/Form.js [hunk 1]
```diff
@@ -121,9 +121,14 @@ class Form {
...
-    // TODO: try to filter down to only submit buttons
     get submitButtons () {
-        return this.form.querySelectorAll(SUBMIT_BUTTON_SELECTOR)
+        return [...this.form.querySelectorAll(SUBMIT_BUTTON_SELECTOR)]
+            .filter((button) => {
+                const content = button.textContent
+                const ariaLabel = button.getAttribute('aria-label')
+                const title = button.title
+                return !/password|show|toggle|reveal|hide/i.test(content + ariaLabel + title)
+            })
```
Comment (GioSensation) [inline src/Form/Form.js:131]: Here I am trying to exclude the little buttons to show and hide passwords.

File: src/Form/Form.js [hunk 2]
```diff
@@ -46,11 +46,16 @@ class Form {
...
         this.getValues = () => {
-            return [...this.inputs.credentials].reduce((output, input) => {
+            const credentials = [...this.inputs.credentials, ...this.inputs.emailNew].reduce((output, input) => {
                 const subtype = getInputSubtype(input)
                 output[subtype] = input.value || output[subtype]
                 return output
             }, {username: '', password: ''})
+            if (credentials.emailNew && !credentials.username) {
+                credentials.username = credentials.emailNew
+            }
```
Comment (GioSensation) [inline src/Form/Form.js:56]: If we don't have a username, let's try and save the email if available.

Comment (jonathanKingston) [reply src/Form/Form.js:56]: Can you add this as a comment :)

File: src/Form/Form.js (hunk 1 above)
Comment (jonathanKingston) [reply src/Form/Form.js:136]: Please add a comment for this 👍🏻

**Review by jonathanKingston — COMMENTED** [review]
LGTM! can you add the two comments please and file a follow up to investigate if we can clean up those selectors?

**Pull request merged**